solutions/swb-reference/integration-tests/tests/isolated/sshKeys/delete.test.ts

```diff
@@ -44,28 +43,52 @@ describe('Delete Key Pair negative tests', () => {
       }
     ];
 
-    beforeEach(() => {
-      invalidSshKeyId = `sshkey-0000000000000000000000000000000000000000000000000000000000000000`;
+    describe('when key does not exist', () => {
+      let invalidSshKeyId: string;
+      beforeEach(() => {
+        invalidSshKeyId = `sshkey-0000000000000000000000000000000000000000000000000000000000000000`;
+      });
+
+      test.each(testBundle)('it throws 403', async (testCase) => {
+        const { username, session: sessionFunc, projectId: projectIdFunc } = testCase;
+        const session = sessionFunc();
+        const projectId = projectIdFunc();
+
+        console.log(`as ${username}`);
+
+        try {
+          await session.resources.projects.project(projectId).sshKeys().sshKey(invalidSshKeyId).purge();
+        } catch (e) {
+          checkHttpError(
+            e,
+            new HttpError(404, {
+              error: 'Not Found',
+              message: `Key ${invalidSshKeyId} does not exist`
+            })
+          );
+        }
+      });
     });
 
-    test.each(testBundle)('within a valid project', async (testCase) => {
-      const { username, session: sessionFunc, projectId: projectIdFunc } = testCase;
-      const session = sessionFunc();
-      const projectId = projectIdFunc();
-
-      console.log(`as ${username}`);
-
-      try {
-        await session.resources.projects.project(projectId).sshKeys().sshKey(invalidSshKeyId).purge();
-      } catch (e) {
-        checkHttpError(
-          e,
-          new HttpError(404, {
-            error: 'Not Found',
-            message: `Key ${invalidSshKeyId} does not exist`
-          })
-        );
-      }
+    describe('when provided key id is invalid', () => {
+      let invalidSshKeyId: string;
+      beforeEach(() => {
+        invalidSshKeyId = `sshkey-malformed`;
+      });
+
+      test.each(testBundle)('it throws 400', async (testCase) => {
+        const { username, session: sessionFunc, projectId: projectIdFunc } = testCase;
+        const session = sessionFunc();
+        const projectId = projectIdFunc();
+
+        console.log(`as ${username}`);
```

Comment:
do you need this console log?

Reply:
yup. it is for testing purposes for the logs to know which session it is running